Graphics/GraphicsAccessories/src/GraphicsAccessories.cpp

```diff
@@ -81,12 +81,14 @@ class TexFormatToViewFormatConverter
         static_assert(TEXTURE_VIEW_RENDER_TARGET == 2, "TEXTURE_VIEW_RENDER_TARGET == 2 expected");
         static_assert(TEXTURE_VIEW_DEPTH_STENCIL == 3, "TEXTURE_VIEW_DEPTH_STENCIL == 3 expected");
         static_assert(TEXTURE_VIEW_UNORDERED_ACCESS == 4, "TEXTURE_VIEW_UNORDERED_ACCESS == 4 expected");
+        static_assert(TEXTURE_VIEW_READ_ONLY_DEPTH_STENCIL == 6, "TEXTURE_VIEW_READ_ONLY_DEPTH_STENCIL == 6 expected");
```

Comment:
Maybe we should add four flags at once? @TheMostDiligent 
```
Depth: Read, Stencil: Read
Depth: Read, Stencil: Write
Depth: Write, Stencil: Read
Depth: Write, Stencil: Write 
```

Reply:
`TEXTURE_VIEW_DEPTH_STENCIL` directly corresponds to `RESOURCE_STATE_DEPTH_WRITE`.
`TEXTURE_VIEW_READ_ONLY_DEPTH_STENCIL` directly corresponds to `RESOURCE_STATE_DEPTH_READ`.

What would "Depth: Read, Stencil: Write" and "Depth: Write, Stencil: Read" correspond to?

Reply:
In Vulkan there are such flags, in d3d12 respectively there is a way to create such combinations
```
 VK_IMAGE_LAYOUT_DEPTH_STENCIL_READ_ONLY_OPTIMAL
 VK_IMAGE_LAYOUT_DEPTH_READ_ONLY_STENCIL_ATTACHMENT_OPTIMAL
 VK_IMAGE_LAYOUT_DEPTH_ATTACHMENT_STENCIL_READ_ONLY_OPTIMAL
 VK_IMAGE_LAYOUT_DEPTH_STENCIL_ATTACHMENT_OPTIMAL
 ```

Reply:
Oh, I didn't know that existed.
I agree that having those may be helpful in general.
However, adding those would require extending `RESOURCE_STATE`, and refactoring that I am afraid I am not quite qualified to do.

If you people decide you want to have all 4 combinations, I would suggest to extract it into separate task, and I will tweak this PR to minimize future changes.

Reply:
Let's wait and see what Egor says. Perhaps it would be better to switch to the option with four states and also consider the possibility of adding dynamic rendering, so that we don't have to add additional fields to the Pipeline State.

Reply:
Yes, I think adding four flags makes sense for completeness.
Dynamic rendering would be and ideal solution here, only that this is a very big change.